src/main/java/com/metsci/laproc/plotting/BasicGraphableData.java

```diff
@@ -10,44 +11,46 @@
  * Represents a continuous line that can be represented on a graph.
  * Created by robinsat on 9/20/2016.
  */
-public class SimpleGraphableData implements GraphableData {
+public class BasicGraphableData implements GraphableData<Double> {
 
     //Fields
     /** The name of this set of data */
     private String name;
     /** The x values in this data set */
-    private double[] xValues;
+    //private double[] xValues;
```

Comment:
There is a lot of commented out stuff in this class that can probably be deleted